doc/lua_api.txt

```diff
@@ -3956,6 +3956,8 @@ Definition tables
         collide_with_objects = true, -- collide with other objects if physical = true
         weight = 5,
         collisionbox = {-0.5, 0.0, -0.5, 0.5, 1.0, 0.5},
+        selectionbox = {0, 0, 0, 0, 0, 0}, -- defaults to collision box dimensions
```

Comment:
Because this _is_ the default setting and what you will see if you dump the property table.